tests/PHPStan/Analyser/nsrt/instanceof-class-string.php

```diff
@@ -32,7 +32,7 @@ public function doBar(Foo $foo, Bar $bar): void
 		if ($foo instanceof $class) {
 			assertType(self::class, $foo);
 		} else {
-			assertType('InstanceOfClassString\Foo~InstanceOfClassString\Bar', $foo);
+			assertType('InstanceOfClassString\Foo', $foo);
```

Comment:
If I call `doFar($foo, $superBar)` with superBar a class extending bar, 
$foo is only `Foo~SuperBar` but can still being Bar.